medication statements/README.md

```diff
@@ -0,0 +1,358 @@
+# Medication Statement Mapping
+
+## XML HL7 > JSON FHIR
+
+A FHIR `MedicationStatement` is mapped from a XML HL7 `MedicationStatement` containing an `EhrSupplyAuthorise` component.</br>
+When `ehrSupplyAuthorise` is referenced, it refers `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyAuthorise`</br>
+When `ehrSupplyDiscontinue` is referenced, it refers to find the first matching `ehrSupplyDiscontinue` where `ehrSupplyDiscontinue / reversalOf / priorMedicationRef / id [@root]` matching `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyAuthorise / id [@root]` in the 'ehrExtract'</br>
+When `ehrSupplyPrescribe` is referenced `MedicationStatement / component [@typeCode="COMP"] / ehrSupplyPrescribe`
+When `EhrExtract` is referenced, it refers to the parent `EhrExtract` in the XML.</br>
+When `PracticeCode` is referenced, it refers to the losing practice ODS code.
```

Comment:
I like this definitions block.

Reply:
I added a bit of whitespace around the sentences, as I was finding it a bit hard to read.